<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <item>
       <layout class="QVBoxLayout" name="ConditionsLayout">
        <property name="leftMargin">
         <number>60</number>
        </property>
        <property name="rightMargin">
         <number>60</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="InitCondLabelLayout">
          <item>
           <widget class="QLabel" name="InitialConditionsLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Initial conditions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ChooseInitCond">
            <item>
             <property name="text">
              <string>Cartesian</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Keplerian</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="InitCondStack">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="KeplerianPage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="PositionLayout">
              <item>
               <widget class="QLabel" name="PositionLabel">
                <property name="text">
                 <string>Initial position:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="PosSpinBoxX">
                <property name="decimals">
                 <number>8</number>
                </property>
                <property name="minimum">
                 <double>-300000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>300000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="PosSpinBoxY">
                <property name="decimals">
                 <number>8</number>
                </property>
                <property name="minimum">
                 <double>-300000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>300000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="PosSpinBoxZ">
                <property name="decimals">
                 <number>8</number>
                </property>
                <property name="minimum">
                 <double>-300000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>300000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="VelocityLayout">
              <item>
               <widget class="QLabel" name="VelocityLabel">
                <property name="text">
                 <string>Initial velocity:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="VelSpinBoxX">
                <property name="decimals">
                 <number>8</number>
                </property>
                <property name="minimum">
                 <double>-300000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>300000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="VelSpinBoxY">
                <property name="decimals">
                 <number>8</number>
                </property>
                <property name="minimum">
                 <double>-300000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>300000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="VelSpinBoxZ">
                <property name="decimals">
                 <number>8</number>
                </property>
                <property name="minimum">
                 <double>-300000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>300000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="CartesianPage">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="EccLayout">
              <item>
               <widget class="QLabel" name="EccLabel">
                <property name="text">
                 <string>Ecc</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="EccSpinBox">
                <property name="decimals">
                 <number>8</number>
                </property>
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="SemMajAxLayout">
              <item>
               <widget class="QLabel" name="SemMajAxLabel">
                <property name="text">
                 <string>Sem. maj. axis</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="SemMajAxSpinBox">
                <property name="decimals">
                 <number>8</number>
                </property>
                <property name="minimum">
                 <double>-300000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>300000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="IncLayout">
              <item>
               <widget class="QLabel" name="IncLabel">
                <property name="text">
                 <string>Inc</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="IncSpinBox">
                <property name="decimals">
                 <number>8</number>
                </property>
                <property name="minimum">
                 <double>-300000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>300000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="RiAscNodeLayout">
              <item>
               <widget class="QLabel" name="RiAscNodeLabel">
                <property name="text">
                 <string>Right asc. node</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="RiAscNodeSpinBox">
                <property name="decimals">
                 <number>8</number>
                </property>
                <property name="minimum">
                 <double>-300000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>300000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="ArgOfPerLayout">
              <item>
               <widget class="QLabel" name="ArgOfPerLabel">
                <property name="text">
                 <string>Arg. of per.</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="ArgOfPerSpinBox">
                <property name="decimals">
                 <number>8</number>
                </property>
                <property name="minimum">
                 <double>-300000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>300000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="TrueAnomLayout">
              <item>
               <widget class="QLabel" name="TrueAnomLabel">
                <property name="text">
                 <string>True anom.</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="TrueAnomSpinBox">
                <property name="decimals">
                 <number>8</number>
                </property>
                <property name="minimum">
                 <double>-300000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>300000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="StartTimeLayout">
          <item>
           <widget class="QLabel" name="StartTimeLabel">
            <property name="text">
             <string>Start time:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="StartTimeSpinBox">
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="EndTimeLayout">
          <item>
           <widget class="QLabel" name="EndTimeLabel">
            <property name="text">
             <string>End time:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="EndTimeSpinBox">
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="TimeStepsLayout">
          <item>
           <widget class="QLabel" name="TimeStepsLabel">
            <property name="text">
             <string>Time steps:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="TimeStepsSpinBox">
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="IntegratorLayout">
          <item>
           <widget class="QLabel" name="IntegratorLabel">
            <property name="text">
             <string>Integrator:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ChooseIntegrator">
            <item>
             <property name="text">
              <string>Euler</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Verlet</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RK4</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="SimulateButton">
          <property name="text">
           <string>Simulate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="OutputLayout">
        <item>
         <widget class="QLabel" name="OutputLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Output values</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="OutputConsole">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionasdasd">
   <property name="text">
    <string>asdasd</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ChooseInitCond</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>InitCondStack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
